core/include/scipp/core/except.h

```diff
@@ -88,6 +84,12 @@ template <class A, class B> void contains(const A &a, const B &b) {
     throw except::NotFoundError("Expected " + to_string(a) + " to contain " +
                                 to_string(b) + ".");
 }
+template <class A, class B> void includes(const A &a, const B &b) {
```

Comment:
Why use `includes` rather than `contains`?

Reply:
Previously we had a double-meaning of `contains`:
- `bool contains(Dim)`;
- `bool contains(Dimensions);`
This change removes the double meaning, and using naming consistent with the standard library.